index.html

```diff
@@ -0,0 +1,789 @@
+      </p>
+      <button type="button" class="grey-green-bg explore__and__newsletter-item-button">Explore JavaScript &amp;
+        Frameworks</button>
+    </article>
+    <article class="explore__and__newsletter-item">
+      <small class="eani-small">a smashing guide to</small>
+      <a class="grey-orange" href="https://www.smashingmagazine.com/guides/css-layout/">
+        <h1 class="grey-orange eani-h1">CSS Layout</h1>
+      </a>
+      <p>
+        CSS Layout has been transformed over the past years. We have you covered with all you need to master this new
+        world. If you want to understand the technical details, or learn how to use new CSS creatively, these guides can
+        help.
+      </p>
+      <button class="explore__and__newsletter-item-button grey-orange-bg" type="button">Explore CSS
+        Layout</button>
+    </article>
+    <article class="explore__and__newsletter-item">
+      <small class="eani-small">a smashing guide to</small>
+      <a class="grey-purple" href="https://www.smashingmagazine.com/guides/performance/">
+        <h1 class="grey-purple eani-h1">Web Performance</h1>
+      </a>
+      <p>
+        Ensuring your site loads quickly is important. The subject touches almost every part of web design and
+        development, from your choice of images to the performance of your web server. In this guide we bring together
+        information that can help you build a fast site, and keep it fast.
+      </p>
+      <button class="explore__and__newsletter-item-button grey-purple-bg" type="button">Explore
+        Performance</button>
+    </article>
+    <div class="newsletter__cat">
+      <div class="newsletter__cat-image">
+        <img src="assets/newsletter.png" alt="">
+      </div>
+      <div class="newsletter__cat-content">
+        <a class="ncc-a" href="https://www.smashingmagazine.com/the-smashing-newsletter/">
+          <h1 class="ncc-h1">Email Newsletter</h1>
+        </a>
+        <p class="ncc-p">
+          Every second Tuesday, we send an email newsletter with useful techniques on front-end and UX. Subscribe and
+          get <strong>Smart Interface Design Checklists PDF</strong> in your inbox.
+        </p>
+      </div>
+      <div class="newsletter__cat-submit">
+        <input class="newsletter__cat-submit-input" type="email" placeholder="Your email ">
+        <button class="newsletter__cat-submit-button" type="button">Meow!</button>
+      </div>
+
+      <div class="newsletter__cat-last-text">
+        <div class="newsletter__cat-last-text-wrapper">
+          <p>
+            Front-end, UX and design. 2× a month.
+          </p>
+          <p>
+            You can always unsubscribe with just 1 click.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="divider">
+    <a class="divider-a" href="https://www.smashingmagazine.com/guides/">see all guides<i
+        class="fas fa-long-arrow-alt-right divider-i"></i></a>
+  </div>
+  <div class="footer">
```

Comment:
Please use a proper `<footer>` semantic tag instead of only a `<div>`. You may change `div` for `footer`, or wrap your footer `<div>` around a `<footer>` tag if you so prefer.

Reply:
Thanks for the feedback. I just added the semantic tags you told me.